### ISSUE TITLE
BR/FR: JSON export does not show correct City/Country for switches

### ISSUE THREAD

**Issue by rasssta** [inex/IXP-Manager#567]
##### ISSUE TYPE

Bug Report / Feature Request

##### OS

Ubuntu 18.04.3 LTS
IXP Manager v5.1.0

##### SUMMARY

We have multiple IXPs in 2 countries: Denmark and Sweden. But we have no way to individually assign city and country for these IXPs and their locations, so they all get "Stockholm, Sweden" in the JSON export, which obviously cause some confusion.

To make it even more interesting our Copenhagen IXP is also present in Sweden so some switches are located in Sweden but we have decided that that IXP should be assigned to Denmark as that's the main part of the IXP.

Examples from our current JSON:
* Copenhagen IXP located in Denmark.
```
        {
            "shortname": "STHIX - Copenhagen",
            "name": "Stockholm Internet eXchange AB",
            "country": "SE",
```

* A switch at STHIX Copenhagen, Global Connect, in Denmark:
```
            "switch": [
                {
                    "id": 20,
                    "name": "sw1-gc1",
                    "colo": "GlobalConnect",
                    "city": "Stockholm",
                    "country": "SE",
```

* A switch at STHIX Copenhagen, Tele2 Malmö, in Sweden:
```
                {
                    "id": 26,
                    "name": "sw1-kv1",
                    "colo": "Tele2 KVG",
                    "city": "Stockholm",
                    "country": "SE",
```

##### STEPS TO REPRODUCE

Our JSON export: https://portal.sthix.net/api/v4/member-export/ixf/1.0

##### EXPECTED RESULTS

I would like the possibility...

* to assign a city and country to each Infrastructure
* to assign a city and country to each Facility, so the switches in the Facility can use that data to know where they are located

It would then give me the following result:

* Copenhagen IXP located in Denmark.
```
        {
            "shortname": "STHIX - Copenhagen",
            "name": "Stockholm Internet eXchange AB",
            "country": "DK",
```

* A switch at STHIX Copenhagen, Global Connect, in Denmark:
```
            "switch": [
                {
                    "id": 20,
                    "name": "sw1-gc1",
                    "colo": "GlobalConnect",
                    "city": "Copenhagen",
                    "country": "DK",
```

* A switch at STHIX Copenhagen, Tele2 Malmö, in Sweden:
```
                {
                    "id": 26,
                    "name": "sw1-kv1",
                    "colo": "Tele2 KVG",
                    "city": "Malmö",
                    "country": "SE",
```

##### IMPORTANCE
Low prio. Noticed it when a customer looked in IXPDB and noticed that the bad data.

**Comment by barryo** [comment]
Some background details on the schema for location:

* https://github.com/euro-ix/json-schemas/pull/26
* https://github.com/euro-ix/json-schemas/pull/34

Right now the IX location is taken from `.env` but this would be better placed in the infrastructure database so we will add that as a todo on this issue.

Ideally consumers of this information would take location details for switches from the PeeringDB facility. But I see we actually hard-coded that to the same `.env` details. We'll update this also for city and country details in IXP Manager's location/facilities database.

**Comment by barryo** [comment]
Fixed via inex/IXP-Manager#578
